roles/satellite/tasks/satellite.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: "Install Satellite with default parameters"
+  command: > 
+    satellite-installer 
+      --scenario satellite
```

Comment:
again, I believe you are looking at the older docs - please check the 6.2 docs:
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/installation_guide/installing_satellite_server#performing_initial_configuration_sat_server_manual